sssom/util.py

```diff
@@ -465,72 +465,88 @@ def dataframe_to_ptable(df: pd.DataFrame, priors=None, inverse_factor: float = 0
     Returns:
         List of rows
     """
-
     if not priors:
         priors = [0.02, 0.02, 0.02, 0.02]
-
-    logging.warning(
-        f"Priors given ({priors}), but not being used by dataframe_to_ptable() method."
-    )
+    else:
+        logging.warning(
+            f"Priors given ({priors}), but not being used by dataframe_to_ptable() method."
+        )
 
     df = collapse(df)
     rows = []
     for _, row in df.iterrows():
-        s = row[SUBJECT_ID]
```

Comment:
should go without saying, one letter variable names are bad because they are not conveying any information to the reader